TST: DeprecationWarning from Jupyter failing CI

Example log: https://github.com/spacetelescope/jdaviz/actions/runs/3430932531/jobs/5718475689

```
ImportError while loading conftest '/home/runner/work/jdaviz/jdaviz/conftest.py'.
../../conftest.py:1: in <module>
    from jdaviz import __version__
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/jdaviz/__init__.py:10: in <module>
    from jdaviz.app import *  # noqa
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/jdaviz/app.py:8: in <module>
    from ipywidgets import widget_serialization
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/ipywidgets/__init__.py:25: in <module>
    from .widgets import *
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/ipywidgets/widgets/__init__.py:4: in <module>
    from .widget import Widget, CallbackDispatcher, register, widget_serialization
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/ipywidgets/widgets/widget.py:19: in <module>
    from ipykernel.comm import Comm
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/ipykernel/__init__.py:5: in <module>
    from .connect import *  # noqa
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/ipykernel/connect.py:11: in <module>
    import jupyter_client
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/jupyter_client/__init__.py:8: in <module>
    from .asynchronous import AsyncKernelClient  # noqa
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/jupyter_client/asynchronous/__init__.py:1: in <module>
    from .client import AsyncKernelClient  # noqa
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/jupyter_client/asynchronous/client.py:8: in <module>
    from jupyter_client.client import KernelClient
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/jupyter_client/client.py:22: in <module>
    from .connect import ConnectionFileMixin
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/jupyter_client/connect.py:27: in <module>
    from jupyter_core.paths import jupyter_data_dir
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/jupyter_core/paths.py:176: in <module>
    deprecation(
../../.tox/py310-test-alldeps-cov/lib/python3.10/site-packages/jupyter_core/utils/__init__.py:83: in deprecation
    warnings.warn(message, DeprecationWarning, stacklevel=stacklevel + 1)
E   DeprecationWarning: Jupyter is migrating its paths to use standard platformdirs
E   given by the platformdirs library.  To remove this warning and
E   see the appropriate new directories, set the environment variable
E   `JUPYTER_PLATFORM_DIRS=1` and then run `jupyter --paths`.
E   The use of platformdirs will be the default in `jupyter_core` v6
```

--- a/tox.ini
+++ b/tox.ini
@@ -14,6 +14,7 @@ isolated_build = true
 # Suppress display of matplotlib plots generated during docs build
 setenv =
     MPLBACKEND=agg
+    JUPYTER_PLATFORM_DIRS=1
     devdeps: PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/scipy-wheels-nightly/simple
 
 # Pass through the following environment variables which may be needed for the CI
@@ -63,6 +64,7 @@ extras =
 
 commands =
     devdeps: pip install -U -i https://pypi.anaconda.org/astropy/simple astropy --pre
+    jupyter --paths
     pip freeze
     !cov: pytest --pyargs jdaviz {toxinidir}/docs {posargs}
     cov: pytest --pyargs jdaviz {toxinidir}/docs --cov jdaviz --cov-config={toxinidir}/setup.cfg {posargs}

